htdocs/_view/CouchDBPassThrough.php

```diff
@@ -0,0 +1,21 @@
+<?php
+ini_set("memory_limit", "2G");
+// Load config file and ensure paths are correct
+set_include_path(
+    get_include_path() . ":../../../project/libraries:../../../php/libraries"
+);
+// Ensures the user is logged in, and parses the config file.
+require_once "NDB_Client.class.inc";
+require_once 'CouchDB.class.inc';
+$client = new NDB_Client();
+$client->initialize("../../project/config.xml");
+
+$cdb = CouchDB::singleton();
+
+$view = $_GET['view'];
+$params = $_GET;
+unset($params['view']);
+
+$result = $cdb->queryView("DQG-2.0", $view, $params, true);
+print_r($result);
```

Comment:
is print_r right? It's printing a PHP array? shouldn't it be print json_encode?

Also, this file should use autoload.php instead of set/get_include_path and require_once..